src/test/java/picard/util/IntervalListToBedTest.java

```diff
@@ -14,57 +15,41 @@
     private static final String TEST_DATA_DIR = "testdata/picard/util/";
     private final File INTERVAL_LIST = new File(TEST_DATA_DIR, "interval_list_to_bed_test.interval_list");
     private final File UNSORTED_INTERVAL_LIST = new File(TEST_DATA_DIR, "unsorted_interval_list_to_bed_test.interval_list");
-    private final File BED_FILE      = new File(TEST_DATA_DIR, "interval_list_to_bed_test.bed");
-
-    @Test
-    public void testConvertToBed() throws Exception {
-        final IntervalListToBed program = new IntervalListToBed();
-        final File tmp = File.createTempFile("interval_list_to_bed_test_output", ".bed");
-        tmp.deleteOnExit();
-
-        final String[] args = {
-                "INPUT=" + INTERVAL_LIST.getAbsolutePath(),
-                "OUTPUT=" + tmp.getAbsolutePath(),
-                "SCORE=333"
+    private final File BED_FILE = new File(TEST_DATA_DIR, "interval_list_to_bed_test.bed");
+    private final File NO_SORT_BED_FILE = new File(TEST_DATA_DIR, "no_sort_interval_list_to_bed_test.bed");
```

Comment:
Any chance you can write 1-2 lines of comments about the properties of these BED files (ex chr5/chr4 are out of karyotype order)?  That way we don't have to look at the test files to know (but can look to verify).